docs/virtual_machines/tekton_tasks.md

```diff
@@ -111,6 +111,14 @@ KubeVirt-specific Tekton tasks, which are focused on:
 All these tasks can be used for creating [pipelines](https://github.com/tektoncd/pipeline/blob/main/docs/pipelines.md).
 TTO is creating multiple example pipelines, e.g.:
 
-- [Windows 10 installer](https://github.com/kubevirt/tekton-tasks-operator/blob/main/data/tekton-pipelines/okd/windows10-installer.yaml) - Pipeline will prepare a template and Windows 10 datavolume. User has to provide a link to working Windows 10 iso file.
+- [Windows 10 installer](https://github.com/kubevirt/tekton-tasks-operator/blob/main/data/tekton-pipelines/okd/windows10-installer.yaml) - Pipeline will prepare a template and Windows 10 datavolume. User has to provide a link to working Windows 10 iso file. 
```

Comment:
nit: remove the trailing whitespace

Reply:
Should we link the documentation of the pipeline, e.g. https://github.com/kubevirt/kubevirt-tekton-tasks/tree/main/examples/pipelines/windows10-installer , instead of the piepline directly, here?